contrib/pkg/createcluster/create.go

```diff
@@ -458,6 +499,52 @@ func (o *Options) GenerateObjects() ([]runtime.Object, error) {
 	return result, err
 }
 
+func (o *Options) addAdoptionInfo(cd *hivev1.ClusterDeployment) ([]runtime.Object, error) {
+	objectsToCreate := []runtime.Object{}
+	cd.Spec.Installed = true
+
+	kubeconfigBytes, err := ioutil.ReadFile(o.AdoptAdminKubeConfig)
+	if err != nil {
+		return objectsToCreate, err
+	}
+
+	adminKubeconfigSecret := &corev1.Secret{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      fmt.Sprintf("%s-adopted-admin-kubeconfig", cd.Name),
+			Namespace: cd.Namespace,
+		},
+		Data: map[string][]byte{
+			"kubeconfig":     kubeconfigBytes,
+			"raw-kubeconfig": kubeconfigBytes,
```

Comment:
Where does "raw-kubeconfig" come from?

Reply:
I am honestly not sure what that's doing there, just that it's what our controllers / installmanager uploads today and I wanted to make sure to match it. It appears to be identical to what goes into the other key.

Reply:
I ask because I don't see the installmanager adding that and I don't see it in the example secrets that I looked at.

Reply:
Hmm I'm getting it with my secrets on a cluster created by hive.

Reply:
I found where this is done now.
https://github.com/openshift/hive/blob/a057c9fcff2fd64fc2c7555b522b3109263f4a12/pkg/controller/clusterdeployment/clusterdeployment_controller.go#L985
I'm still trying to figure out *why* it is done.

Reply:
The raw one is the original created by the installer. The other is the one fix the additional CAs added in.

Reply:
Ohhhh right, that's it. I remember now.